test/e2e/e2e_leaderelection_test.go

```diff
@@ -95,9 +99,20 @@ func TestLeaderElection(t *testing.T) {
 	s2.Client = clientSet
 	s2.DeschedulingInterval = 5 * time.Second
 	s2.LeaderElection.LeaderElect = true
+	s2.LeaderElection.RetryPeriod = metav1.Duration{
+		Duration: time.Second,
+	}
 	s2.ClientConnection.Kubeconfig = os.Getenv("KUBECONFIG")
 	s2.PolicyConfigFile = "./policy_leaderelection_b.yaml"
 
+	// Delete the descheduler lease
+	err = clientSet.CoordinationV1().Leases("kube-system").Delete(ctx, "descheduler", metav1.DeleteOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		t.Fatalf("Unable to remove kube-system/descheduler lease: %v", err)
+	} else {
+		t.Logf("Removed kube-system/descheduler lease")
```

Comment:
a bit misleading since not found doesn't actually remove

Reply:
You are right. Updated. Thx.